pegasus/sites.v3/code.org/views/display_language_lightbox.haml

```diff
@@ -1,4 +1,4 @@
-%script{type: "text/javascript", src: "/js/bootstrap.min.js"}
+%script{:defer => "defer", type: "text/javascript", src: "/js/bootstrap.min.js"}
```

Comment:
This is relied upon by any calls to `.modal()`, I see one call in this file below on [line 40](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/views/display_language_lightbox.haml#L40-L40). This call is within `$(document).ready`, so as far as I can tell this _should_ occur after the defer-loaded javascript, so should be okay.